src/vars/lists/assoc_lists.md

```diff
@@ -36,4 +36,31 @@ var/list/L = list(fizz = "buzz", money = 100)
 
 >**Notice:** See how the keys don't have double quotes around them?
 
-TODO: Associated lists and their loops, cursed numerical keys, and list->assoc list conversion funtimes
+## Associated List Loops
+
+Looping through associated lists is fairly straightforward. You can either loop through the list items,
+
+```dm
+var/list/exlist = list(fizz = "buzz", money = 100)
+var/i
+
+for(i in exlist)
+	world << "[i] = [exlist[i]]"
+```
+
+which will print out the key while using it to grab value associated with it, or you can loop through the array indicies.
+
+```dm
+var/j
+var/k
+
+for(j = 1, j <= exlist.len, j++)
```

Comment:
```suggestion
for(j = 1, j <= length(exlist), j++)
```
bad bad no never

Reply:
<img width="185" alt="ripdmdoc" src="https://user-images.githubusercontent.com/45720148/122656908-837d8000-d124-11eb-98b7-50340b0000ad.png">
Gotta delete the DM Reference now.

Reply:
yes.